.eleventy.js

```diff
@@ -89,16 +127,26 @@ module.exports = function (config) {
   let markdownLibrary = markdownIt({
     html: true,
     breaks: true,
-    linkify: true,
+    linkify: false
```

Comment:
why no links? they're useful for deep linking on docs pages to headers of specific sections

Reply:
I think this is already old. Closing PR.